doc/nrf/app_dev/wifi_coex/index.rst

```diff
@@ -135,12 +145,41 @@ To configure the hardware for a PTA inside the nRF700x series chips:
    *  ``802154_rpmsg`` for applications having support for 802.15.4, but not for Bluetooth.
    *  ``hci_rpmsg`` for application having support for Bluetooth, but not for 802.15.4.
 
-.. _ug_radio_cx_generic_3_wire_hw_if:
+#. Make sure the following Kconfig options are enabled:
+
+   * :kconfig:option:`CONFIG_MPSL_CX`
+   * :kconfig:option:`CONFIG_MPSL_CX_NRF700X`
 
-Generic three wire Coexistence hardware interface
-=================================================
+   Please note that if a ``nordic,nrf700x-coex`` compatible node is present in the device tree and :kconfig:option:`CONFIG_MPSL_CX` is set to ``y``, :kconfig:option:`MPSL_CX_NRF700X` will be selected by default.
 
-To configure the hardware for a PTA using a 3-wire interface:
+.. _ug_radio_mpsl_cx_generic_3wire:
+
+Generic three wire coexistence
+==============================
+
+Please refer to :ref:`ug_radio_coex_mpsl_cx_based` for general requirements of this implementation.
+
+Hardware description
+--------------------
+
+The generic three wire interface consists of signals listed in the table below.
+The *Pin* is a generic pin name of a PTA, identified rather by its function.
+The *Direction* is from point of view of the SoC running SR protocol.
+The *DT property* is the name of devicetree node property that configures connection between SoC running SR protocol and nRF700x.
+
+.. table:: Generic three wire coexistence protocol pins
+
+   ============  =========  =================================  ==============
+   Pin           Direction  Description                        DT property
+   ============  =========  =================================  ==============
+   REQUEST       Out        Request signal                     req-gpios
+   PRIORITY      Out        SR transaction direction TX or RX  pri-dir-gpios
+   GRANT         In         Grant signal                       grant-gpios
+   ============  =========  =================================  ==============
+
+
+Enabling nRF700x Wi-Fi coexistence
```

Comment:
This should be `Enabling generic three wire coexistence`

Reply:
fixed